### DISCUSSION TITLE
Progress capture lidR catalog

### DISCUSSION THREAD

**Discussion by bdmiller42** [r-lidar/lidR#476] [Q&A] [answered]
Hi, I'm using an non-local computer (HPC) to normalize a lidR LAS-catalog using a DTM. I get a certain amount of processing time that needs to be requested and the allocation I am allowed to make is not long enough to process the entire catalog. I am using the ORIGINALFILENAME option for output files. Based on the modified date of the output files, it is not proceeding through the entire catalog from where it finished, but starting from the beginning of the catalog again. I'm essentially processing in circles and it is wasting processor time for me to normalize the point clouds.

Is there some way to save the state of processing so it is easy to pick back up where it finished the last time? Did I miss this in the docs? Should I just remove the files that have been processed from the LAS catalog before beginning normalization again?

Any help would be appreciated.

**Comment by Jean-Romain** [comment, marked as answer]
With the next version 3.2.0 (that will be released soon, but you can install from github), yes it is easy to restart where you left. Let say you processed 300 chunks then:

```r
opt_restart(ctg) <- 300
```
